2024-03-22 10:07:49,005 INFO    MainThread:109749 [wandb_setup.py:_flush():76] Current SDK version is 0.16.3
2024-03-22 10:07:49,006 INFO    MainThread:109749 [wandb_setup.py:_flush():76] Configure stats pid to 109749
2024-03-22 10:07:49,006 INFO    MainThread:109749 [wandb_setup.py:_flush():76] Loading settings from /home/souvik/.config/wandb/settings
2024-03-22 10:07:49,006 INFO    MainThread:109749 [wandb_setup.py:_flush():76] Loading settings from /mnt/d/WSL/Flight-Fare-Prediction/wandb/settings
2024-03-22 10:07:49,007 INFO    MainThread:109749 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-03-22 10:07:49,007 INFO    MainThread:109749 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-03-22 10:07:49,008 INFO    MainThread:109749 [wandb_init.py:_log_setup():526] Logging user logs to /mnt/d/WSL/Flight-Fare-Prediction/wandb/run-20240322_100748-Data Types Count Plot/logs/debug.log
2024-03-22 10:07:49,008 INFO    MainThread:109749 [wandb_init.py:_log_setup():527] Logging internal logs to /mnt/d/WSL/Flight-Fare-Prediction/wandb/run-20240322_100748-Data Types Count Plot/logs/debug-internal.log
2024-03-22 10:07:49,009 INFO    MainThread:109749 [wandb_init.py:init():566] calling init triggers
2024-03-22 10:07:49,009 INFO    MainThread:109749 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {}
2024-03-22 10:07:49,010 INFO    MainThread:109749 [wandb_init.py:init():591] re-initializing run, found existing run on stack: Data Types Count Plot
2024-03-22 10:07:49,014 INFO    MainThread:109749 [wandb_run.py:_finish():1970] finishing run ghsouvikprof/Flight Fare/Data Types Count Plot
2024-03-22 10:07:49,015 ERROR   MainThread:109749 [jupyter.py:save_history():437] Run pip install nbformat to save notebook history
2024-03-22 10:07:49,016 INFO    MainThread:109749 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-03-22 10:07:49,016 INFO    MainThread:109749 [wandb_init.py:_jupyter_teardown():454] cleaning up jupyter logic
2024-03-22 10:07:49,017 INFO    MainThread:109749 [wandb_run.py:_atexit_cleanup():2210] got exitcode: 0
2024-03-22 10:07:49,017 INFO    MainThread:109749 [wandb_run.py:_restore():2193] restore
2024-03-22 10:07:49,018 INFO    MainThread:109749 [wandb_run.py:_restore():2199] restore done
2024-03-22 10:07:58,282 INFO    MainThread:109749 [wandb_run.py:_footer_history_summary_info():3866] rendering history
2024-03-22 10:07:58,286 INFO    MainThread:109749 [wandb_run.py:_footer_history_summary_info():3898] rendering summary
2024-03-22 10:07:58,297 INFO    MainThread:109749 [wandb_run.py:_footer_sync_info():3825] logging synced files
2024-03-22 10:07:59,918 INFO    MainThread:109749 [wandb_init.py:init():616] starting backend
2024-03-22 10:07:59,918 INFO    MainThread:109749 [wandb_init.py:init():620] setting up manager
2024-03-22 10:07:59,924 INFO    MainThread:109749 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-03-22 10:07:59,925 INFO    MainThread:109749 [wandb_init.py:init():628] backend started and connected
2024-03-22 10:07:59,948 INFO    MainThread:109749 [wandb_run.py:_label_probe_notebook():1295] probe notebook
2024-03-22 10:07:59,949 INFO    MainThread:109749 [wandb_run.py:_label_probe_notebook():1305] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-03-22 10:07:59,949 INFO    MainThread:109749 [wandb_init.py:init():720] updated telemetry
2024-03-22 10:07:59,996 INFO    MainThread:109749 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2024-03-22 10:08:04,804 INFO    MainThread:109749 [wandb_run.py:_on_init():2262] communicating current version
2024-03-22 10:08:08,204 INFO    MainThread:109749 [wandb_run.py:_on_init():2271] got version response upgrade_message: "wandb version 0.16.4 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2024-03-22 10:08:08,205 INFO    MainThread:109749 [wandb_init.py:init():804] starting run threads in backend
2024-03-22 10:08:14,304 INFO    MainThread:109749 [wandb_run.py:_console_start():2241] atexit reg
2024-03-22 10:08:14,304 INFO    MainThread:109749 [wandb_run.py:_redirect():2096] redirect: wrap_raw
2024-03-22 10:08:14,305 INFO    MainThread:109749 [wandb_run.py:_redirect():2161] Wrapping output streams.
2024-03-22 10:08:14,305 INFO    MainThread:109749 [wandb_run.py:_redirect():2186] Redirects installed.
2024-03-22 10:08:14,307 INFO    MainThread:109749 [wandb_init.py:init():847] run started, returning control to user process
2024-03-22 10:11:48,817 INFO    MainThread:109749 [wandb_setup.py:_flush():76] Current SDK version is 0.16.3
2024-03-22 10:11:48,817 INFO    MainThread:109749 [wandb_setup.py:_flush():76] Configure stats pid to 109749
2024-03-22 10:11:48,818 INFO    MainThread:109749 [wandb_setup.py:_flush():76] Loading settings from /home/souvik/.config/wandb/settings
2024-03-22 10:11:48,818 INFO    MainThread:109749 [wandb_setup.py:_flush():76] Loading settings from /mnt/d/WSL/Flight-Fare-Prediction/wandb/settings
2024-03-22 10:11:48,818 INFO    MainThread:109749 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-03-22 10:11:48,819 INFO    MainThread:109749 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-03-22 10:11:48,820 INFO    MainThread:109749 [wandb_init.py:_log_setup():526] Logging user logs to /mnt/d/WSL/Flight-Fare-Prediction/wandb/run-20240322_101148-Top 5 Airline Pie Chart/logs/debug.log
2024-03-22 10:11:48,820 INFO    MainThread:109749 [wandb_init.py:_log_setup():527] Logging internal logs to /mnt/d/WSL/Flight-Fare-Prediction/wandb/run-20240322_101148-Top 5 Airline Pie Chart/logs/debug-internal.log
2024-03-22 10:11:48,821 INFO    MainThread:109749 [wandb_init.py:_jupyter_setup():472] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7fd221c20390>
2024-03-22 10:11:48,821 INFO    MainThread:109749 [wandb_init.py:init():566] calling init triggers
2024-03-22 10:11:48,822 INFO    MainThread:109749 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {}
2024-03-22 10:11:48,822 INFO    MainThread:109749 [wandb_init.py:init():591] re-initializing run, found existing run on stack: Data Types Count Plot
2024-03-22 10:11:48,825 INFO    MainThread:109749 [wandb_run.py:_finish():1970] finishing run ghsouvikprof/Flight Fare/Data Types Count Plot
2024-03-22 10:11:48,826 INFO    MainThread:109749 [wandb_run.py:_atexit_cleanup():2210] got exitcode: 0
2024-03-22 10:11:48,826 INFO    MainThread:109749 [wandb_run.py:_restore():2193] restore
2024-03-22 10:11:48,827 INFO    MainThread:109749 [wandb_run.py:_restore():2199] restore done
2024-03-22 10:11:57,339 INFO    MainThread:109749 [wandb_run.py:_footer_history_summary_info():3866] rendering history
2024-03-22 10:11:57,340 INFO    MainThread:109749 [wandb_run.py:_footer_history_summary_info():3898] rendering summary
2024-03-22 10:11:57,353 INFO    MainThread:109749 [wandb_run.py:_footer_sync_info():3825] logging synced files
